From ccbccdaeefc980d4a512a96b1e918c2cfde0e00d Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Tue, 20 Dec 2011 23:28:18 +0000
Subject: cpan/Test-Harness: add NAME headings in modules with POD

This fixes the Debian Lintian warning about missing NAME sections in
manpages.

Bug: https://rt.cpan.org/Ticket/Display.html?id=73399
Bug-Debian: http://bugs.debian.org/650451

Patch-Name: fixes/manpage_name_Test-Harness.diff
---
 cpan/Test-Harness/lib/TAP/Harness/Beyond.pod |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/cpan/Test-Harness/lib/TAP/Harness/Beyond.pod b/cpan/Test-Harness/lib/TAP/Harness/Beyond.pod
index ed77e13..7281508 100644
--- a/cpan/Test-Harness/lib/TAP/Harness/Beyond.pod
+++ b/cpan/Test-Harness/lib/TAP/Harness/Beyond.pod
@@ -1,3 +1,7 @@
+=head1 NAME
+
+Test::Harness - Beyond make test
+
 =head1 Beyond make test
 
 Test::Harness is responsible for running test scripts, analysing
